isis/appdata/translations/voyager.trn

```diff
@@ -0,0 +1,204 @@
+Group = MissionPhaseName
```

Comment:
You could also put this at `ISIS3/isis/src/voyager/translations/` I think there was one other mission we did this with, and it seems slightly preferable to this location to me.